src/magicbox/ResultPreviewsManager.ts

```diff
@@ -173,17 +174,25 @@ export class ResultPreviewsManager {
     this.suggestionsPreviewContainer = null;
   }
 
+  private getIdForPreviews() {
+    return `coveo-previews-for-${this.lastDisplayedSuggestion.id}`;
+  }
+
   private buildPreviewContainer() {
     return $$(
       'div',
       {
-        className: 'coveo-preview-container'
+        className: 'coveo-preview-container',
+        id: this.getIdForPreviews()
       },
       (this.resultPreviewsHeader = $$('div', {
-        className: 'coveo-preview-header'
+        className: 'coveo-preview-header',
+        role: 'status'
```

Comment:
curious about what it does add?

Reply:
I don't know if there's a particular accessibility use that checks for the `status` role, but it seemed relevant to indicate the header as being a `status` since it only contains text that gets updated whenever the results change.

Reply:
Oh, do you hear it when it changes? with voiceover?

Reply:
Unfortunately I couldn't hear it with the voiceover on MacOS, but I imagine that some other accessibility software might be handling it.